README.md

```diff
@@ -32,8 +32,22 @@ Bumping patch for version 1.2.4
 v1.2.3
 v1.2.4
 v1.2.5
+
+> gitversion --prefix v show
+v1.2.5
 ```
 
+### Auto
+
+Auto is a special field that will determine the proper field to bump
+based on the contents of the last commit message.  It looks for anything matching:
+
+> major: or [major]
+> minor: or [minor]
+> patch: or [patch]
+
```

Comment:
For some reason the example is being displayed all in one line.
Perhaps add a line between each:

">major: or [major]"

">minor: or [minor]"

">patch: or [patch]"